tests/lax_numpy_test.py

```diff
@@ -2303,6 +2302,31 @@ def testFull(self, shape, fill_value_dtype, out_dtype, rng_factory):
     self._CheckAgainstNumpy(np_fun, jnp_fun, args_maker)
     self._CompileAndCheck(jnp_fun, args_maker)
 
+  @parameterized.named_parameters(jtu.cases_from_list(
+       {"testcase_name":
+           "_shape={}_n={}_axis={}_prepend={}_append={}".format(
+           jtu.format_shape_dtype_string(shape, dtype),
+           n, axis, prepend, append),
+        "shape": shape, "dtype": dtype, "n": n, "axis": axis,
+        "prepend": prepend, "append": append,
+        "rng_factory": jtu.rand_default}
+       for shape, dtype in _shape_and_dtypes(nonempty_nonscalar_array_shapes, default_dtypes)
+       for n in [0, 1, 2]
+       for axis in list(range(-len(shape), max(1, len(shape))))
+       for prepend in [None, 1, jtu._rand_dtype(np.random.rand, shape, dtype)]
+       for append in [None, 1, jtu._rand_dtype(np.random.rand, shape, dtype)]
+       ))
+  def testDiff(self, shape, dtype, n, axis, prepend, append, rng_factory):
+    rng = rng_factory(self.rng())
+    args_maker = lambda: [rng(shape, dtype)]
+    if prepend is None and append is None: np_fun = lambda x: np.diff(x, n=n, axis=axis)
```

Comment:
@jakevdp I couldn't figure out a better way to call np diff because it checks for ```np._NoValue ``` instead of ```None```. So passing ```None``` gives an error